func.go

```diff
@@ -60,7 +60,7 @@ func RegisterLibFunc(fptr any, handle uintptr, name string) {
 //	int64 <=> int64_t
 //	float32 <=> float
 //	float64 <=> double
-//	struct <=> struct (WIP - darwin only)
+//	struct <=> struct (darwin amd64/arm64, linux amd64/arm64)
```

Comment:
Did you mean to push the Linux changes? I thought we were keeping them separate

Reply:
sorry missed a commit, fixing.